routing.rst

```diff
@@ -192,6 +192,11 @@ Matching HTTP Methods
 By default, routes match any HTTP verb (``GET``, ``POST``, ``PUT``, etc.)
 Use the ``methods`` option to restrict the verbs each route should respond to:
 
+.. note::
+
+    ``HEAD`` verb is considered as ``GET`` verb. Please use both ``GET`` and ``HEAD`` 
+    in verb conditionned routes.
```

Comment:
I think this is not descriptive enough. How about something like:
> `HEAD` is considered equivalent of `GET`, thus routes configured to accept `GET` will always accept `HEAD` too.

Reply:
This is cool I think, but it's important to note that head is changed
It may change behaviour if you hardcode on top of HEAD verb